### PR TITLE
Add packaging to required dependencies

### LINKED ISSUE #6: Missing packaging dependency
- [X] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

The `packaging` package is not listed in this feedstock as a dependency even though it is required by the [parameter handler code](https://github.com/openforcefield/openff-toolkit/blob/eedd8ac39dad79eab7e5f027dca063b6622f30ca/openff/toolkit/typing/engines/smirnoff/parameters.py#L1906).

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

# Name                    Version                   Build  Channel
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
astunparse                1.6.3              pyhd8ed1ab_0    conda-forge
black                     20.8b1                     py_1    conda-forge
blosc                     1.21.0               h046ec9c_0    conda-forge
bzip2                     1.0.8                hc929b4f_4    conda-forge
c-ares                    1.17.1               h0d85af4_1    conda-forge
ca-certificates           2020.12.5            h033912b_0    conda-forge
certifi                   2020.12.5        py39h6e9494a_1    conda-forge
click                     7.1.2              pyh9f0ad1d_0    conda-forge
dataclasses               0.8                pyhc8e2a94_1    conda-forge
decorator                 4.4.2                      py_0    conda-forge
fftw                      3.3.9           nompi_h02cd531_101    conda-forge
hdf5                      1.10.6          nompi_hc5d9132_1114    conda-forge
isort                     5.8.0              pyhd8ed1ab_0    conda-forge
krb5                      1.17.2               h60d9502_0    conda-forge
libblas                   3.9.0                8_openblas    conda-forge
libcblas                  3.9.0                8_openblas    conda-forge
libcurl                   7.75.0               h8ef9fac_0    conda-forge
libcxx                    11.1.0               habf9029_0    conda-forge
libedit                   3.1.20191231         h0678c8f_2    conda-forge
libev                     4.33                 haf1e3a3_1    conda-forge
libffi                    3.3                  h046ec9c_2    conda-forge
libgfortran               5.0.0           9_3_0_h6c81a4c_20    conda-forge
libgfortran5              9.3.0               h6c81a4c_20    conda-forge
liblapack                 3.9.0                8_openblas    conda-forge
libnghttp2                1.43.0               h07e645a_0    conda-forge
libopenblas               0.3.12          openmp_h54245bb_1    conda-forge
libssh2                   1.9.0                h52ee1ee_6    conda-forge
llvm-openmp               11.1.0               hda6cdc1_0    conda-forge
mdtraj                    1.9.5            py39hb6ec638_1    conda-forge
mock                      4.0.3            py39h6e9494a_1    conda-forge
mypy_extensions           0.4.3            py39h6e9494a_3    conda-forge
ncurses                   6.2                  h2e338ed_4    conda-forge
networkx                  2.5                        py_0    conda-forge
numexpr                   2.7.3            py39h4d6be9b_0    conda-forge
numpy                     1.20.1           py39h3c955ea_0    conda-forge
openff-forcefields        1.3.0              pyh44b312d_0    conda-forge
openff-toolkit-base       0.9.1              pyhd8ed1ab_1    conda-forge
openmm                    7.5.0           py39hbb05945_6_apple    conda-forge
openssl                   1.1.1j               hbcf498f_0    conda-forge
pandas                    1.2.3            py39h4d6be9b_0    conda-forge
pathspec                  0.8.1              pyhd3deb0d_0    conda-forge
pip                       21.0.1             pyhd8ed1ab_0    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pytables                  3.6.1            py39hd07922a_3    conda-forge
python                    3.9.2           h2502468_0_cpython    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python_abi                3.9                      1_cp39    conda-forge
pytz                      2021.1             pyhd8ed1ab_0    conda-forge
readline                  8.1                  h05e3726_0    conda-forge
regex                     2021.3.17        py39h89e85a6_0    conda-forge
scipy                     1.6.1            py39h056f1c0_0    conda-forge
setuptools                49.6.0           py39h6e9494a_3    conda-forge
six                       1.15.0             pyh9f0ad1d_0    conda-forge
smirnoff-plugins          0+untagged.1.g1ed7bc6.dirty           dev_0    <develop>
smirnoff99frosst          1.1.0              pyh44b312d_0    conda-forge
snappy                    1.1.8                hb1e8313_3    conda-forge
sqlite                    3.35.2               h44b9ce1_0    conda-forge
tk                        8.6.10               h0419947_1    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
typed-ast                 1.4.2            py39hcbf5805_0    conda-forge
typing_extensions         3.7.4.3                    py_0    conda-forge
tzdata                    2021a                he74cb21_0    conda-forge
wheel                     0.36.2             pyhd3deb0d_0    conda-forge
xmltodict                 0.12.0                     py_0    conda-forge
xz                        5.2.5                haf1e3a3_1    conda-forge
zlib                      1.2.11            h7795811_1010    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : smirnoff-plugins
    active env location : /Users/XXX/miniconda3/envs/smirnoff-plugins
            shell level : 1
       user config file : /Users/XXX/.condarc
 populated config files : /Users/XXX/miniconda3/.condarc
                          /Users/XXX/.condarc
          conda version : 4.9.2
    conda-build version : not installed
         python version : 3.8.6.final.0
       virtual packages : __osx=10.16=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /Users/XXX/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /Users/XXX/miniconda3/pkgs
                          /Users/XXX/.conda/pkgs
       envs directories : /Users/XXX/miniconda3/envs
                          /Users/XXX/.conda/envs
               platform : osx-64
             user-agent : conda/4.9.2 requests/2.25.1 CPython/3.8.6 Darwin/20.2.0 OSX/10.16
                UID:GID : 501:20
             netrc file : /Users/XXX/.netrc
           offline mode : False
```
</details>

### PR DESCRIPTION
**Pull request by mattwthompson**
Resolves #6

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

**Review by j-wags — APPROVED** [review]
Looks good to me. Should we bump the build number, or does conda-forge handle that automatically?

**Comment by mattwthompson** [comment]
Whoops - thought I pushed that earlier.

> does conda-forge handle that automatically

It seems not, and actually our previously build have been numbered 1 when they should have been numbered 0. As far as I know, it's not problematic as long as we +1 it when we should.

On the checklist for a reason - oops!

**Comment by mattwthompson** [comment]
I have no clue how the SHA changed or why this recipe was passing in #5

```
$ curl -sL https://github.com/openforcefield/openforcefield/archive/0.9.1.tar.gz | openssl sha256
(stdin)= 17e6d7f5f1673fa5ebabce74c07c5017fa132dba2bebf17fde314fb1de00d167

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-About openff-toolkit
-====================
+About openff-toolkit-split
+==========================
```
Comment (mattwthompson) [inline README.md:2]: Heh, interesting bit caught here by the bot

**Pull request merged**